.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+# Description
+
+Include a summary of the changes made and mention the issue it fixes. 
+
+Fixes # (include the issue number here)
+
+# Checklist:
+
+- [ ] My code follows the [guidelines](https://github.com/ankitdobhal/Awesome-Python-Scripts/blob/master/Contribution.md) of this project.
+- [ ] I have commented my code.
+- [ ] I have performed a self-review of my own code.
+- [ ] I have added all the reviewers: @ankitdobhal and @SKAUL05
```

Comment:
Remove this line not necessary.
Add:
-  I have followed all the appropriate naming conventions.